### PR TITLE
ClearlyDefinedUploadCommand: Output the number of successful uploads

### PR DESCRIPTION
**Pull request by fviernau**
For providing a short summary.

Signed-off-by: Frank Viernau <frank.viernau@here.com>

### CODE REVIEW

File: cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt [hunk 1]
```diff
@@ -142,24 +142,26 @@ class ClearlyDefinedUploadCommand : CliktCommand(
...
+                    println("was uploaded successfully:\n${summary.url}")
 
-                else -> println("was uploaded successfully:\n${summary.url}")
+                    uploadedCurationsCount++
+                }
             }
         }
 
-        if (error) {
+        println("Uploaded $uploadedCurationsCount / ${uploadableCurations.size} curations.")
```
Comment (sschuberth) [inline cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt:162]: Nit: Mind spelling out "/" as "of" like we do in the `ReporterCommand`?

File: cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt [hunk 2]
```diff
@@ -142,24 +142,26 @@ class ClearlyDefinedUploadCommand : CliktCommand(
...
-                null -> {
-                    println("failed to be uploaded.")
-                    error = true
-                }
+                null -> println("failed to be uploaded.")
+                else -> {
+                    println("was uploaded successfully:\n${summary.url}")
 
-                else -> println("was uploaded successfully:\n${summary.url}")
+                    uploadedCurationsCount++
```
Comment (sschuberth) [inline cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt:157]: Nit: Mind changing this to a prefix increment, like `++uploadedCurationsCount`? Before you ask, it's just a matter of personal taste, and I prefer the prefix increment if the return of the previous value is not needed, like here.

File: cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt (hunk 1 above)
Comment (fviernau) [reply cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt]: Done. I've also added 'Successfully' like in the `ReporterCommand`.

File: cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt (hunk 2 above)
Comment (fviernau) [reply cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt]: Done.

**Pull request merged**